src/components/ItemDetail/use/editItem.ts

```diff
@@ -0,0 +1,63 @@
+import apis, { Owner } from '/@/lib/apis'
+import useMe from '/@/use/me'
+import { useStore } from '/@/store'
+
+const useEditItem = (): {
+  editItem: (payload: {
+    itemID: number
+    userID: number
+    ownInfo: Owner
+    rentalable: boolean
+    count: number
+  }) => Promise<void>
+} => {
+  const editItem = async (payload: {
+    itemID: number
+    userID: number
+    ownInfo: Owner
+    rentalable: boolean
+    count: number
+  }): Promise<void> => {
+    const store = useStore()
+    const { id: meID } = useMe()
+    const id = payload.userID ?? meID.value
```

Comment:
```suggestion
    const id = payload.userID
```
これ同値じゃない？`number`は`undefined`でも`null`でもないから